src/components/raw-capacity-card/raw-capacity-card.tsx

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright contributors to the ibm-storage-odf-console project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from 'react';
+import * as _ from 'lodash';
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import { ChartDonut, ChartLabel } from '@patternfly/react-charts';
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+  usePrometheusPoll,
+} from '@console/dynamic-plugin-sdk/internalAPI';
+import { humanizeBinaryBytes } from '../../humanize';
+import { 
+  FlASHSYSTEM_QUERIES,
+  StorageDashboardQuery,
+ } from '../../constants/queries';
+import {
+  useK8sWatchResource,
+} from "@console/dynamic-plugin-sdk/api";
+import {GetFlashSystemResource} from '../../constants/resources';
+import {parseMetricData} from '../../selectors/promethues-utils';
+import { StorageInstanceKind } from '../../types';
+import './raw-capacity-card.scss';
+
+const colorScale = ['#0166cc', '#d6d6d6'];
+
+const RawCapacityCard: React.FC<any> = (props)  => {
+  const { t } = useTranslation();
+  const [data, loaded, loadError] = useK8sWatchResource<StorageInstanceKind>(GetFlashSystemResource(props?.match?.params?.name, props?.match?.params?.namespace));
```

Comment:
is both names are same? (props?.match?.params?.namespace and data?.[0]?.metadata.name)

Reply:
I am trying to understand why watcher used here